src/markdown-pages/add-ons/ekco.md

```diff
@@ -113,6 +114,12 @@ To avoid race conditions, manually run the ekco-reboot service's shutdown script
 /opt/ekco/shutdown.sh
 ```
 
+### Internal Load Balancer
+
+Ekco 0.11.0+ can maintain an internal load balancer forwarding all traffic from host port 6444 to one of the Kubernetes API server pods.
```

Comment:
We should be consistent with case. Are we using EKCO or ekco?

Reply:
EKCO